products/views.py

```diff
@@ -47,6 +48,13 @@ def get(self, request, product_title):
                         'star_rating'  : review.star_rating
                         } for review in reviews]
 
-        review_info.append(product.review_set.aggregate(avg=Avg('star_rating'),count=Count('star_rating')))
+        product_rate = product.review_set.aggregate(avg=Avg('star_rating'),count=Count('star_rating'))
+
+        page_size = 10
+        limit = page_size * page
+        offset = limit - page_size
```

Comment:
상수화는 보통 대문자로 지정해줍니다.
```suggestion
        PAGE_SIZE = 10
        limit = page_size * page
        offset = limit - page_size
```

Reply:
아래 컨벤션만 맞춰주세요!

Reply:
깜빡했습니당.. 수정 했습니다.